components/DynamicForm.js

```
@@ -390,6 +390,40 @@ const DynamicForm = ({ prefillData }) => {
+    const updatedData = { ...formData };
+    if (!updatedData.apps[index]) return;
+
+    updatedData.apps[index].whitelist = [];
+    setFormData(updatedData);
+    reset(updatedData);
+  };
+
+  const handleRemoveWhitelistFromApp = (index) => {
+    const updatedData = { ...formData };
+    if (updatedData.apps[index] && 'whitelist' in updatedData.apps[index]) {
+      delete updatedData.apps[index].whitelist;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)